cmd/influx/config.go

```diff
@@ -104,40 +78,18 @@ func (b *cmdConfigBuilder) cmdCreate() *cobra.Command {
 }
```

Comment:
need to remove the active flag here if you decide to keep it implictly setting this to active when creating. I think that implicit switch is not something we want here, I'd rather see the active flag from the user take care of that instead of of forcing it every create...

If you decide to keep it implicit, this flag needs to be removed.

Reply:
@jademcgough what are your thoughts here?

Reply:
I create a bug for this one, let's not make any behavior change in this pr. https://github.com/influxdata/influxdb/issues/17534